contents/blog/how-we-turned-clickhouse-into-our-eventmansion.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2021-11-08
+title: How we turned ClickHouse into our event mansion
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+Author: joe-martin
+featuredImage: ../images/blog/clickhouse-announcement.png
+featuredImageType: full
+---
+
+Recently, PostHog was invited to speak at OSA Con 2021, an open source analytics conference organised by Altinity. It was a fantastic opportunity to talk to the community, alongside other engineers and leaders from companies such as eBay, Facebook and Rudderstack. 
+
+Topics ranged from deep dives into reverse ETL processes to discussions about broad trends in the data engineering sector. At PostHog, we chose to talk about our experience [migrating PostHog from Postgres to ClickHouse](https://posthog.com/blog/clickhouse-announcement) — why we did it, how it went and what impact it had. ClickHouse is a technology we’re heavily invested in, so we wanted to share some points from our presentation here too. 
+
+## The night before ClickHouse
+
+The original version of PostHog ran on Postgres. In fact, the _very_ first version was an MVP created by our co-founder, Tim Glaser, over the course of [four intense weeks at YCombinator](https://posthog.com/blog/inflated-risk-seems-riskier). 
+
+These early versions were straightforward web apps which relied entirely on Django ORM for queries and ran on Postgres because of its capacity to handle relational data. They had easy, one-click deployments set up via Heroku. 
```

Comment:
NIT: `These early versions` suggests multiple which is confusing if it means multiple different ventures or versions of posthog.